Documentation/subcommands/begin.md

```diff
@@ -38,17 +38,33 @@ The `arch` and `os` labels are filled in with the architecture and operating
 system of the machine acbuild is running on. If this is undesirable, the labels
 can be modified or removed with the `acbuild label` command.
 
+## Starting with a pre-existing rootfs
+
+If the user has a rootfs they wish to use in the ACI, perhaps produced by a
+tool like [buildroot](http://buildroot.org/), a directory can be passed to
+begin. The contents of the directory will be copied into the ACI, and the ACI
+will have a manifest identical to when the begin command is not passed a
+directory.
+
+When specifying something on the local filesystem to the begin command, the
+path to it _must_ start with `.`, `~`, or `/`. As an example, if the directory
+is in the current directory, then instead of passing in `buildroot/output`,
+what would be passed in is `./buildroot/output`.
+
+## Starting state
```

Comment:
Perhaps "Starting with a pre-existing ACI", and promote this above the rootfs one?